config/template.config

```diff
@@ -8,7 +8,7 @@ includeConfig "${projectDir}/nextflow.config"
 
 // Inputs/parameters of the pipeline
 params {
-    algorithm = ['stats', 'collectwgsmetrics'] // 'fastqc', 'stats', 'collectwgsmetrics', 'bamqc'
+    algorithm = ['stats', 'collectwgsmetrics'] // 'fastqc', 'stats', 'collectwgsmetrics', 'windows', 'bamqc'
```

Comment:
**(question | non-blocking)** As we're adding more algorithms, I feel it's a bit ambiguous to say just `windows` or `stats`. We may want to specifically spell out the tool names to avoid confusion or come up with a better way to specify algorithms. Any thoughts?